unifiedconfig/src/main/java/io/camunda/unifiedconfig/Camunda.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.unifiedconfig;
+
+public class Camunda {}
```

Comment:
:wrench: Let's start the habit of documenting the classes and properties in this module. Can you add a short java doc for this class?